pcdsdevices/pim.py

```diff
@@ -193,7 +193,7 @@ def __init__(self, prefix, *, name, prefix_det=None, prefix_zoom=None,
         if prefix_zoom:
             self._prefix_zoom = prefix_zoom
         else:
-            self._prefix_zoom = self.prefix_start+'CLZ:01:'
+            self._prefix_zoom = self.prefix_start+'CLZ:01'
```

Comment:
I assume this works because there are no further sub-pv's like `<...>:CLZ:01.RBV` (records?   I don't know the right term).  

Do the detector prefixes work?  They're defined identically but only the zoom prefix needed the change